demos/smart_classroom_demo/cpp_gapi/main.cpp

```diff
@@ -8,12 +8,12 @@
 #include <monitors/presenter.h>
 #include <utils/performance_metrics.hpp>
 #include <ie_iextension.h>
+#include <stream_source.hpp>
```

Comment:
why is it changed ? are there any dependencies in headers below ?

Reply:
I have added `cpp_gapi` folder to `common`. This file isn't a part of one sample now. 
** Changed to `#include <utils/stream_source.hpp>` for understanding.